src/components/common/ColonyFunding/ColonyFunding.tsx

```diff
@@ -1,9 +1,36 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { usePageHeadingContext, useSetPageHeadingTitle } from '~context';
 import FundsTable from '~frame/v5/pages/FundsPage/partials/FundsTable';
+import { formatText } from '~utils/intl';
 
 const displayName = 'common.ColonyFunding';
 
-const ColonyFunding = () => <FundsTable />;
+const ColonyFunding = () => {
+  const { setBreadcrumbs } = usePageHeadingContext();
+  useSetPageHeadingTitle(formatText({ id: 'incomingFundsPage.title' }));
+
+  useEffect(() => {
+    setBreadcrumbs([
+      {
+        key: 'teams',
+        // @todo: replace with actual teams
+        dropdownOptions: [
+          {
+            label: 'All teams',
+            href: '/teams',
+          },
+        ],
+        selectedValue: '/teams',
+      },
+    ]);
+
+    return () => {
+      setBreadcrumbs([]);
+    };
+  }, [setBreadcrumbs]);
```

Comment:
Can you add a hook for breadcrumbs that would work the same way as `useSetPageHeadingTitle` ?

Reply:
Hook added